### ISSUE TITLE
[Bug][A11Y] Separate label from trait, state, value and error message

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1188]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

.

### App version

0.22.0

### App build number

1764327028

### App build type

alpha

### Describe the issue

After using Voice Control, I've noticed that some components have incorrect `accessibilityLabel` because Trait, `accessibilityValue `or `errorDescription `are concatened in the label

For example: Input label has "Label, Vide" as an accessible label, which is not relevant for Voice Control users, who need to speak the visible name of the component to interact with (for example : "touch Label")

<img width="387" height="146" alt="" src="https://github.com/user-attachments/assets/7be4a5f9-5b06-4f65-b442-33af55335480" />

### Expected Behavior

Separate `accessibilityLabel` from fake trait, state (deactivated), `accessibilityValue `and `errorDescription`

Fake trait, deactivated state, `accessibilityValue `and `errorDescription` can be kept together as the global `accessibilityValue`

Here is the detail for each impacted component:

- checkbox: error description should be used in value
- radio button: the fake "radio button" trait and error desc should be used in value
- switch: fake "switch" trait and deactivated state should be used in value
- textInput: empty status should be used in value, error message and loader status also

### Accessibility settings

Voice Control
